packages/common/src/authentication/AuthContext.tsx

```diff
@@ -101,22 +100,23 @@ export const AuthProvider: FC = ({ children }) => {
   const i18n = useI18N();
   const defaultLanguage = useDefaultLanguage();
   const { mutateAsync, isLoading: isLoggingIn } = useGetAuthToken();
-  const { token: cookieToken, store: cookieStore, user } = getAuthCookie();
-  const [localStore, setLocalStore] = useState<Store | undefined>(cookieStore);
-  const [localToken, setLocalToken] = useState<string | undefined>(cookieToken);
-  const storeId = localStore?.id ?? '';
+  const authCookie = getAuthCookie();
+  const [cookie, setCookie] = useState<AuthCookie | undefined>(authCookie);
```

Comment:
changed to cookie - the `user` was outside of local state and so was a bit weird. 
was using the `user` to trigger the logged out modal, which meant after login the newly assigned user wasn't being passed to the logged out modal, and you get prompted again.